packages/react-components/src/components/DevicePermissions/DomainPermissions.tsx

```diff
@@ -19,6 +19,10 @@ export interface CommonDomainPermissionsProps {
    * Name of application calling experience is in.
    */
   appName: string;
+  /**
+   * Type of the Domain Permissions component.
+   */
+  type: 'request' | 'denied';
```

Comment:
nit: Does this need to be a required prop? Will we not provide a default value like setting to "request" by default?

Reply:
We could, and open to, but I want to be explicit first until we have a strong argument to provide a default

Reply:
Not having one might suggest this should be two seperate component API instead:
`CameraDomainPermissionsRequest` and `CameraDomainPermissionsDenied` (that under the hood just use this).
Because this is in beta and we can adjust we'll see how we use this and get an API review then can adjust accordingly